### DISCUSSION TITLE
Show off your panel!

### DISCUSSION THREAD

**Discussion by Jas-SinghFSU** [Jas-SinghFSU/HyprPanel#3] [Announcements]
If you've got a cool theme you'd like to share, nows your chance :D. Share some screenshots of your setup!

**Comment by ashebanow** [comment]
Here's an early cut at customizing. Still have a long way to go. Using catppucin mocha theme for most stuff, but I haven't wired the theme stuff to HyprPanel yet.

Overall though, I'm very happy with HyprPanel so far. It looks much more polished than waybar stuff I had before, and it seems a lot snappier as well. Nice work!

![Screenshot_27-Jul_17-23-07_28101](https://github.com/user-attachments/assets/ea8bbb7b-fb52-4099-8f01-702022009818)

**Reply by Jas-SinghFSU** [reply]
Excited to see it finished!

**Reply by Acisama** [reply]
How did you color your workspace dots?

**Reply by ashebanow** [reply]
Sorry I didn't respond your question earlier, I missed it. The dots are colored as active/inactive. I don't think there is a way to color based on the type of workspace but not 100% sure.

**Comment by rubiin** [comment]
Here is mine 
![image](https://github.com/user-attachments/assets/2027411a-427b-4cb4-89d8-a20e101fe7a5)

**Comment by RuhanShafi** [comment]
The only thing I miss from Waybar is being able to see the CPU temperature on the bar.
![base](https://github.com/user-attachments/assets/c13e44e7-178b-4305-a49b-6681e67607d9)

**Reply by Jas-SinghFSU** [reply]
It's there, just add the `cputemp` module.

**Reply by RuhanShafi** [reply]
Not working for me. Tried replacing the "CPU" module, but still no cookie

**Reply by Jas-SinghFSU** [reply]
Are you on latest of master? Running `git pull`

**Comment by greed-d** [comment]
A shot at nordic for hyprpanel

![image](https://github.com/user-attachments/assets/785315e0-992a-42fd-a1cd-33db0f8d96b4)

**Comment by NaiveInvestigator** [comment]
late to the game lol

i finally got to dis after berating Jas a lot ;-;

very very sorry lol to him 
tysm for all the help!!

![image](https://github.com/user-attachments/assets/4b4f6d0a-4c12-4b6f-a66f-e68b90bb35f2)

**Comment by onefish2** [comment]
After a few weeks of tweaking. I am really loving Hyprpanel!!
![snapshot_2025-01-06_19-47-02](https://github.com/user-attachments/assets/30cbac8d-e797-4f03-bc96-0e961f5558c9)

**Comment by se7en-x230** [comment]
![image](https://github.com/user-attachments/assets/53397eb9-dbcc-49c2-bd9e-00276bb5b01c)

is there a way to add custom modules like in waybar?
calling a script that outputs infos in the panel?

**Reply by greed-d** [reply]
there are various premade module, see if there is something you need

**Reply by Jas-SinghFSU** [reply]
There is now: https://hyprpanel.com/configuration/custom_modules.html

**Comment by yunusey** [comment]
<img width="2880" height="1800" alt="image" src="https://github.com/user-attachments/assets/8f570726-6ac5-4cdf-b5ba-96d496ff1544" />



I've recently switched from my AGS config to HyprPanel as it offered more functionality and was much easier to customize. Currently, HyprPanel doesn't support control buttons in the `bar/media` widget (and I am not sure if it ever will), which is why I have a [fork of the repo](https://github.com/yunusey/HyprPanel/tree/yunusey-custom) that:

- Adds control buttons (prev, play/pause, and next) to the `bar/media` widget (153317cdf7902ee4c8f13b1c55a43f5fe237ce2b),
- Allows the user to customize right and middle clicks for the `ShortcutButtons`. I added this feature and now when I left-click on the snapshot button, it runs `hyprshot -m window`, when I right-click, it runs `hyprshot -m output`, and when I middle-click, it runs `hyprshot -m region` (c520d3150ed91e6028310e0752b54ae2867593f8),
- Sets the primary click (left-click) on the `bar/volume` widget to mute and sets the secondary click (right-click) to open the `audiomenu` (d9ad5a4f24b25dec8a41736649623307d6685418).

If anyone of these interest you, you can take a look at the linked particular changes, and patch them directly or with some more changes. If @Jas-SinghFSU would be interested in integrating any of these into the master, I can also open up a PR.

Thanks @Jas-SinghFSU and other maintainers for creating this tool!

**Reply by Glue415** [reply]
As somone thoroughly not smart enough to implement your changes, I hope the creator implements them (at least as options) Great job!

**Reply by crocodilestick** [reply]
What are you running YT music in there? Is it a client or a browser with minimal UI or something? It looks very cool

**Reply by yunusey** [reply]
> As somone thoroughly not smart enough to implement your changes, I hope the creator implements them (at least as options) Great job!

Thanks! I switched to a custom [Quickshell config](https://quickshell.org/) recently, and I am not quite aware of the recent developments of the project, but I hope the maintainers will try their best to make it as customizable as possible. If you would like to check the changes I made, you can take a look at my branch.

**Reply by yunusey** [reply]
> What are you running YT music in there? Is it a client or a browser with minimal UI or something? It looks very cool

Thanks! I am using [th-ch/youtube-music](https://github.com/th-ch/youtube-music) with [catppuccin theme](https://github.com/catppuccin/youtubemusic) with some custom tweaks. You can learn more about theming youtube-music app [here](https://github.com/th-ch/youtube-music?tab=readme-ov-file#themes). As for the fancy UI, the app automatically switches to a minimal view when you decrease the width of the window. If you want to launch it in this minimal view (assuming you are in Hyprland), you can do something like this:

```hyprlang
windowrule = float, title:(com.github.th_ch.youtube_music)
windowrule = center, title:(com.github.th_ch.youtube_music)
windowrule = size 600 800, title:(com.github.th_ch.youtube_music)
```
